archive/2022/README.md

```diff
@@ -19,3 +19,8 @@
 | Marcel Faltus          | Evaluating the impact of the x86 hardware memory ordering on the Apple M1 processor through QEMU                              | BA   | Redha Gouicem                                             | [PDF](summer/docs/bsc_faltus_evaluating_the_impact_of_the_x86_hardware_memory_ordering_on_the_Apple_M1processor_through_QEMU.pdf)                               | [PDF](summer/talks/bsc_faltus_evaluating_the_impact_of_the_x86_hardware_memory_ordering_on_the_Apple_M1processor_through_QEMU.pdf)                               |
 
 ## Winter semester
+| Student                | Title                                                                                                                         | Type | Advisors                                                  | Report/Thesis                                                                                                                                                   | Presentation                                                                                                                                                     |
+| ---------------------- | ----------------------------------------------------------------------------------------------------------------------------- | ---- | --------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Manuel Cardenas        | Multiple-Pose Augmented Learning System                                                                                       | BA   | José Vega                                                 | [PDF](winter/docs/bsc_cardenas_multiple_pose_augmented_learning_system.pdf)                                                                                     | [PDF](winter/talks/bsc_cardenas_multiple_pose_augmented_learning_system.pptx)                                                                                    |
```

Comment:
Is it possible to upload pdf instead of pptx?

Reply:
@manoletre ping?

Reply:
sorry, somehow missed your last messages. In the final presentation I created some videos, which displayed what I developed. I can replace them by pictures and create a pdf, but some information would go missing. As you prefer.